src/writeData/utils/parsePluginsConfigurationText.mjs

```diff
@@ -290,22 +301,28 @@ const getVersion = new Promise((resolve, reject) => {
   }
 })
 
-const parsedPluginsPath =
+const parsedPluginsConfigPath =
   'src/writeData/components/telegrafInputPluginsConfigurationText/'
 
-const fetch = path => {
+const parsedPluginsReadmePath = 'src/writeData/components/telegrafPlugins/'
+
+const fetch = (path, parsedPath) => {
   return new Promise((resolve, reject) => {
     https.get(path, response => {
       let contents = ''
+      if (response.statusCode !== 200) {
```

Comment:
will this ever return a 201 or 204?

Reply:
Probably not. But if it happens, the output in the terminal will show the status code and an error message. Since this script is currently manually triggered, the script will be run again in an attempt to get a 200.

There are no plans to automate this script as part of the pipeline process at the moment. And there are good reasons for that, mostly involving new plugins needing specific assets which are manually updated the first time.